client/app/scripts/stores/app-store.js

```diff
@@ -453,7 +536,18 @@ export class AppStore extends Store {
       setTopology(payload.state.topologyId);
       setDefaultTopologyOptions(topologies);
       selectedNodeId = payload.state.selectedNodeId;
-      controlPipe = payload.state.controlPipe;
+      if (payload.state.controlPipe) {
+        controlPipes = makeOrderedMap(
+          [payload.state.controlPipe.pipeId, payload.state.controlPipe]
```

Comment:
You're right, `[[payload.state.controlPipe.pipeId, payload.state.controlPipe]]` it is.